### ISSUE TITLE
Image Plane XSize YSize not copied to other freecad documents

### ISSUE THREAD

**Issue by Chocolateandmilkwin** [FreeCAD/FreeCAD#12920]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

If you copy and paste an image plane from one freecad document to another, the XSize and YSize is reset to what it automatically sets when you initially import the image file, either between two windows of freecad or between two documents in the same window.
When copy/pasting in the same document it works ok.

### Full version info

```shell
[code]
OS: Windows 11 build 22631
Word size of FreeCAD: 64-bit
Version: 0.21.2.33771 (Git)
Build type: Release
Branch: releases/FreeCAD-0-21
Hash: b9bfa5c5507506e4515816414cd27f4851d00489
Python 3.8.10, Qt 5.15.2, Coin 4.0.1, Vtk 8.2.0, OCC 7.6.3
Locale: Danish/Denmark (da_DK)
Installed mods: 
  * Estimate 0.1.2
[/code]
```

### Subproject(s) affected?

None

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by wwmayer** [comment]
https://github.com/FreeCAD/FreeCAD/issues/11036

**Comment by wwmayer** [comment]
I think the fix must be backported to v0.21

**Comment by luzpaz** [comment]
Apologies, I thought i read somewhere that we aren't making another backport release.

**Comment by wwmayer** [comment]
> Apologies, I thought i read somewhere that we aren't making another backport release.

Why this? IMO, the stable version must be maintained until the development version becomes the new stable version.

**Comment by wwmayer** [comment]
Fixed with 2e025dfca3
